Gemfile.lock

```diff
@@ -824,4 +824,4 @@ DEPENDENCIES
   youtube-dl.rb
 
 BUNDLED WITH
-   1.14.5
+   1.14.6
```

Comment:
Have we standardized on a version of `bundler`?  Setup guide still recommends the outdated 1.10.6: https://github.com/code-dot-org/code-dot-org/blob/staging/SETUP.md#overview